src/index.js

```diff
@@ -50,6 +50,7 @@ export { GvTree } from './molecules/gv-tree';
 export { GvDocumentation } from './organisms/gv-documentation';
 export { GvHeader } from './organisms/gv-header';
 export { GvMenu } from './organisms/gv-menu';
+export { GvNewsletterSubscriptionPage } from './organisms/gv-newsletter-subscription-page';
```

Comment:
Maybe we can go with `GvNewsletterSubscription`? As there is no more `:root` CSS I'm not sure the `Page` term is still right

Reply:
--> It looks like there is already a GvNewsletterSubscription component in APIM, maybe it feels better to rename it to something like GvNewsletterReminder and use GvNewsletterSubscription for the new web component

Reply:
👍 agree

Reply:
it's fixed :)